### PR TITLE
PoC build fix

### PR DESCRIPTION
**Pull request by red2k18**

### CODE REVIEW

**Comment by red2k18** [comment]
@blueandgold I had commented it out as the build was failing due to the error thrown while creating G Suite resource and didn't want this to prevent this error from running integration tests.

However, I took some time and fixed it. Build logs can be found here: https://travis-ci.org/forseti-security/forseti-security/builds/601401774

**Comment by dekuhn** [comment]
@red2k18 Can you outline what the root cause was and how this addresses that problem?

File: integration_tests/fixtures/forseti/main.tf [hunk 1]
```diff
@@ -135,15 +135,17 @@ resource "google_kms_crypto_key" "test-crypto-key" {
...
 provider "gsuite" {
   version     = "~> 0.1"
   credentials = "/workspace/release-silver.json"
   impersonated_user_email = "admin@silver.forsetisecurity.dev"
 }
 
 resource "gsuite_group" "devteam" {
-  email       = "automated-group-creation@silver.forsetisecurity.dev"
-  name        = "automated-group-creation@silver.forsetisecurity.dev"
+  email       = "gsuite-${random_pet.random_name_generator.id}@silver.forsetisecurity.dev"
```
Comment (blueandgold) [inline integration_tests/fixtures/forseti/main.tf]: Can you please explain how this random name would fix the build?  What was the build failure?

File: integration_tests/fixtures/forseti/main.tf [hunk 2]
```diff
@@ -135,15 +135,17 @@ resource "google_kms_crypto_key" "test-crypto-key" {
...
 }
 
+
 provider "gsuite" {
   version     = "~> 0.1"
   credentials = "/workspace/release-silver.json"
   impersonated_user_email = "admin@silver.forsetisecurity.dev"
 }
 
 resource "gsuite_group" "devteam" {
```
Comment (blueandgold) [inline integration_tests/fixtures/forseti/main.tf]: Can you please rename `devteam` to something else?  How about `testingrocks` ?

File: integration_tests/fixtures/forseti/main.tf (hunk 1 above)
Comment (red2k18) [reply integration_tests/fixtures/forseti/main.tf]: Build failed as the group name and email were not randomized. Names need to be randomized as after deleting the resource, new resource cannot be created with the same name for certain period of time.

Comment (blueandgold) [reply integration_tests/fixtures/forseti/main.tf]: Got it.  Thanks for explaining.

**Comment by red2k18** [comment]
Build passed: https://travis-ci.org/forseti-security/forseti-security/jobs/603027053?utm_medium=notification&utm_source=github_status

**Comment by red2k18** [comment]
Fixes #3355

**Pull request merged**